.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,8 @@
+- id: helm_docs
+  args: []
+  description: Uses 'helm-docs' to create documentation from the Helm chart's 'values.yaml' file, and inserts the result into a corresponding 'README.md' file.
+  entry: scripts/helm_docs.sh
+  files: (Chart|requirements)\.yaml$
```

Comment:
I'm really sorry, I blanked. Should probably put `values.yaml` in there too, changes to that will affect documentation.

Reply:
Updated.